src/main/kotlin/no/nav/medlemskap/domene/PersonhistorikkEktefelle.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.medlemskap.domene
+
+data class PersonhistorikkEktefelle(
```

Comment:
Jeg er litt usikker på navnene på denne og PersonhistorikkBarn. Det er på en måte ikke så veldig beskrivende, all den tid det bare inneholder bittelitt data om identer. Men jeg foreslår at vi evt gjør om på det i en egen PR. Hele Personhistorikk-opplegget er kanskje modent for revisjon, inkl de klassene som var der fra før.

Reply:
Ja jeg føler mye kan bli veldig mye enklere og penere den dagen vi kvitter oss med TPS..